dockstore-integration-testing/src/test/java/io/dockstore/client/cli/EntryResourceIT.java

```diff
@@ -46,6 +48,7 @@
 @ExtendWith(SystemStubsExtension.class)
 @ExtendWith(MuteForSuccessfulTests.class)
 @ExtendWith(TestStatus.class)
+@Tag(HoverflyTest.NAME)
```

Comment:
Is there a reason this is tagged when there are no other changes in the file?

Reply:
There's a test in the file that uses hoverfly, but I realized it's an ORCID test so I'm going to move it to ORCIDIT